x11-libs/libdrm/libdrm-9999.ebuild

```diff
@@ -15,7 +15,6 @@ DESCRIPTION="X.Org libdrm library"
 HOMEPAGE="https://dri.freedesktop.org/"
 if [[ ${PV} = 9999* ]]; then
 	SRC_URI=""
-	KEYWORDS="~arm"
```

Comment:
What about to remove the libdrm.ebuild entirely? It's obsolete now since the grate-specific libdrm is now bundled into opentegra, mesa and libvdpau-tegra.

Reply:
This can be improved later on.